src/util/samplebuffer.h

```diff
@@ -199,7 +205,7 @@ namespace std {
 
 // Template specialization of std::swap() for SampleBuffer
 template<>
-inline void swap(::mixxx::SampleBuffer& lhs, ::mixxx::SampleBuffer& rhs) {
+inline void swap(::mixxx::SampleBuffer& lhs, ::mixxx::SampleBuffer& rhs) /* noexcept */ {
```

Comment:
Why is noexcept commented out?

Reply:
Doesn't compile on mac, but probably will in the future.

Reply:
What is the error message. 
Please add an explanation as comment.

Reply:
Done.